chapters/external-repository-identifiers.md

```diff
@@ -238,15 +238,19 @@ Examples:
 
 External Reference Site: [Software Heritage persistent IDentifiers](https://docs.softwareheritage.org/devel/swh-model/persistent-identifiers.html)
 
-### F.4.1 gitbom <a name="gitbom"></a>
+### F.4.2 gitbom <a name="gitbom"></a>
 
-Locator Format: gitoid":"\<git object type>":"\<hash algorithm>":"\<hash value>
-	
-Contextual Example: gitoid:blob:sha1:261eeb9e9f8b2b4b0d119366dda99c6fd7d35c64
+Locator Format: `gitoid":"\<git object type>":"\<hash algorithm>":"\<hash value>`
```

Comment:
Should this start with `"gitoid:"` ? I mean, the `gitoid` part is standard text, it should be in quotes.

Reply:
Thanks. For this line, I was merely copying what was present, and you're correct, "gitoid:" is a literal string.

Reply:
One small note: Locators are not identifiers.  Shouldn't this be 'Identifier Format' ?